docs/data-sources/sql_warehouse.md

```diff
@@ -38,14 +38,14 @@ This data source exports the following attributes:
 * `tags` - Databricks tags all warehouse resources with these tags.
 * `spot_instance_policy` - The spot policy to use for allocating instances to clusters: `COST_OPTIMIZED` or `RELIABILITY_OPTIMIZED`.
 * `enable_photon` - Whether to enable [Photon](https://databricks.com/product/delta-engine).
-* `enable_serverless_compute` - Whether this SQL warehouse is a Serverless warehouse. To use a Serverless SQL warehouse, you must enable Serverless SQL warehouses for the workspace.
+* `enable_serverless_compute` - Whether this SQL warehouse is a serverless SQL warehouse. **For AWS**, Databricks strongly recommends that you always explicitly set this field. If omitted, the default is false for most workspaces. However, if this workspace used the SQL Warehouses API to create a warehouse between September 1, 2022 and April 6, 2023, the default remains the previous behavior which is default to true if the workspace is enabled for serverless and fits the requirements for serverless SQL warehouses. To avoid ambiguity, especially for organizations with many workspaces, Databricks recommends that you always set this field. To use serverless SQL warehouses, if your account was created before October 1, 2021, your organization's owner or account administrator must [accept applicable terms of use](https://docs.databricks.com/sql/admin/serverless.html#accept-terms) before workspaces are enabled for serverless compute. A workspace must meet the [requirements](https://docs.databricks.com/sql/admin/serverless.html#requirements) and might require an update its instance profile role to [add a trust relationship](https://docs.databricks.com/sql/admin/serverless.html#aws-instance-profile-setup). **For Azure**, you must [enable your workspace for serverless SQL warehouse](https://learn.microsoft.com/azure/databricks/sql/admin/serverless). For Azure, if serverless SQL warehouses are disabled for the workspace, the default is `false`. If serverless SQL warehouses are enabled for the workspace, the default is `true`.
```

Comment:
@jxbell let leave only the `Whether this SQL warehouse is a serverless SQL warehouse. ` part here. Because it's a data source we don't need the full description here